cmd/root.go

```diff
@@ -75,11 +75,58 @@ func init() {
 	rootCmd.AddCommand(restoreCmd)
 }
 
-func initConfig() {
-	// find shome directory
-	home, err := os.UserHomeDir()
+// get one of available config path from environment variable, $HOME/.config, $HOME
+func getConfigHomePath() string {
+	if xdg_home := os.Getenv("XDG_CONFIG_HOME"); xdg_home != "" {
```

Comment:
One minor change: can we follow `camelCase` for naming here? (i.e. `xdgHome` instead of `xdg_home`)

Reply:
No need. If you make a change in the same branch that you made these changes in, it will automatically reflect here in this Pull Request. (i.e. https://github.com/kmollee/autosaved/tree/main)

Reply:
In general, it isn't recommended to make PR changes on your `main` branch. Instead, you can do that on a new branch and make a pull request with that.
One of the later steps in this guide may be helpful to you: https://www.freecodecamp.org/news/how-to-make-your-first-pull-request-on-github-3/

Reply:
Thanks for the tips. :)

Reply:
Naming issue has fixed.